# Fly action to be handled by the drone

#Define the goal
geometry_msgs/Pose start_pose
geometry_msgs/Pose target_pose
---
# Define the result
geometry_msgs/Pose final_pose
---
# Define a feedback message
# No intermediate result
